webview-ui/package.json

```diff
@@ -21,7 +21,6 @@
 		"debounce": "^2.1.1",
 		"fast-deep-equal": "^3.1.3",
 		"fzf": "^0.5.2",
-		"lucide-react": "^0.474.0",
```

Comment:
why we remove it

Reply:
I think we're supposed to be using https://github.com/microsoft/vscode-codicons for icon to adhere to the VSCode style guidelines, but I'm not opposed to using Lucide for some things if we like it better. I think there's an open question about how "native" we want to look within VSCode.